content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'GDS File Creation from Template'
+linkTitle: 'Create GDS Files With Our Github Template'
+description: 'The detailed procedure to get GDS from a wokwi template repo'
+weight: 60
+---
+
+
+This guide will help you through the essential steps to set up a GitHub repository, enable actions, and successfully commit your design for TinyTapeout. Follow each step carefully.
+
+## Use the Template Repository
+-------------------------------
+
+Start by using the template repository to create your own copy of the project. This will help you build upon an existing foundation.
+
+![Use Template Repository](images/template.png)
+
+## Enable GitHub Actions
+-------------------------
+
+Enabling GitHub Actions allows automated tasks to run, like testing or building your project. Make sure to activate actions for your repository.
```

Comment:
Better something like: GitHub Actions allow automated tasks to run.  They are enabled by default, but we need to enable them to publish the results.